.github/workflows/documentation.yml

```diff
@@ -1,33 +1,46 @@
-name: Documentation
+name: 'Documentation - Build and Deploy'
```

Comment:
I named it "Documentation" on purpose. Here I tell you why:
![image](https://user-images.githubusercontent.com/20406381/109354194-b81bb200-787d-11eb-90ff-50c303a19bc7.png)

With this workflow every PR has the build step that runs and the deploy step which is skipped. I would like to have the naming clean so the checks that are shown by github on PRs are:
- Documentation / build
- Documentation / deploy
and not
- Documentation - Build and Deploy / Build
- Documentation - Build and Deploy / Deploy

It is more an esthetic problem then a "real" problem

Reply:
Just finished looking into this. Another :-1:  experience with Github Actions for CI for me :disappointed: 

These aren't pull request specific, but the [`Check Suites`](https://docs.github.com/en/rest/reference/checks) results for each Check Run performed from workflows triggered, not jobs. Doesn't seem that you can opt-out/hide job status from pull requests which is a bit annoying..

This will always add noise with the Docs Deploy job always being skipped. I think we should split the workflow separately to keep the deploy job status hidden?

> It is more an aesthetic problem then a "real" problem

I see that now, thanks for pointing it out!

Sadly doesn't appear to be a nice way to  re-label the Check Run name (As in `<workflow name> / <job name>` is non-configurable).